### ISSUE TITLE
onPage should start at 1

### ISSUE THREAD

**Issue by rgroves** [aj-4/100m-startup#1]
https://github.com/aj-4/100m-startup/blob/9015550c2ecd8f04a4c441db33561a442dd7376a/worker/tasks/fetch-github.js#L14

If onPage starts at 0 the results are duplicate between the call with page=0 and page=1. There will be 50 duplicates in your data store.

If you compare the results of these example calls you'll see that is the case:
https://jobs.github.com/positions.json?description=ruby&page=0
https://jobs.github.com/positions.json?description=ruby&page=1

Probably not critical for this tutorial sample app, but pointing it out for good measure. 🙂

It's a small change to make, but if you'd like a PR let me know.

**Comment by join2akshay** [comment]
yes, I simply changed `onPage=0` to  `onPage=1`. It removed duplicate elements from `allJobs`
https://github.com/aj-4/100m-startup/blob/9015550c2ecd8f04a4c441db33561a442dd7376a/worker/tasks/fetch-github.js#L14
